### PR TITLE
fix(readme): fix example in readme

### PR DESCRIPTION
**Pull request by mefellows**
Fixes example in README so that it can be copy/pasted and run.

### CODE REVIEW

**Comment by mefellows** [comment]
This doesn't require the use of babel or anything so I think it's probably a bit nicer - you can just run it with `mocha <filename>` and it should work. Thoughts?

**Comment by tarciosaraiva** [comment]
Thanks @mefellows. Actually you might need babel-register depending on what you doing and the node version you're running. Node v4 implemented a good level of ES6 features and arrow functions are there so I believe it shouldn't be a problem.

As for the README, thanks for fixing that up - didn't realise there was a mistake there with two `it`. In any case please have a look at my comments on the file itself.

File: README.md [hunk 1]
```diff
@@ -70,18 +70,18 @@ More questions about what's involved in Pact? [Read more about it](http://docs.p
...
-import Promise from 'bluebird'
-import request from 'superagent'
-import wrapper from '@pact-foundation/pact-node'
-
-import { default as Pact } from 'pact'
+var path = require('path')
+var chai = require("chai")
+var expect = chai.expect
+var chaiAsPromised = require("chai-as-promised")
+var Promise = require('bluebird')
```
Comment (tarciosaraiva) [inline README.md]: maybe we could remove this... avoid confusion?

File: README.md [hunk 2]
```diff
@@ -70,18 +70,18 @@ More questions about what's involved in Pact? [Read more about it](http://docs.p
...
+var request = require ('superagent')
+var wrapper = require('@pact-foundation/pact-node')
+var Pact = require('pact')
+chai.use(chaiAsPromised);
 
 // great library to spin up the Pact Verifier Server
```
Comment (tarciosaraiva) [inline README.md]: get rid of these comments

File: README.md [hunk 3]
```diff
@@ -129,15 +129,15 @@ describe('Pact', () => {
   })
 
   context('with a single request', () => {
-    it('successfully writes Pact file', (done) => {
+    describe('successfully writes Pact file', (done) => {
```
Comment (tarciosaraiva) [inline README.md]: `describe` should be used in sync mode. Remove the `done`

File: README.md (hunk 2 above)
Comment (mefellows) [inline README.md]: Yeah I agree. It IS a great library though :)

File: README.md (hunk 1 above)
Comment (mefellows) [inline README.md]: Done.

**Comment by mefellows** [comment]
Thanks @tarciosaraiva. I've cleaned up as discussed. I can run the example with Node v4 (which is quite old now and the latest 6.x.x) with pure Mocha goodness.

**Comment by tarciosaraiva** [comment]
Good stuff. Merging in.

**Pull request merged**